EIPS/eip-4834.md

```diff
@@ -112,26 +109,27 @@ interface IDomain is IERC165 {
 
     /// @notice     Get if an account can delete the subdomain with a given name
     /// @dev        This must return `false` if `hasDomain(name)` is `false`.
-    ///             If `getDomain(name)` is a domain, this should return `false` if
-    ///             `getDomain(name).canDeleteSubdomain(msg.sender, this, subdomain)` is `false`.
+    ///             If `getDomain(name)` is a domain implementing the subdomain access control extension, this should return `false` if `getDomain(name).canDeleteSubdomain(msg.sender, this, subdomain)` is `false`.
     /// @param      updater The account that may or may not be able to delete a subdomain
     /// @param      name The subdomain to delete
     /// @return     Whether an account can delete the subdomain
     function canDeleteDomain(address updater, string memory name) external view returns (bool);
 }
 ```
 
+As per [EIP-165](./eip-165.md), `supportsInterface(type(IDomain).interfaceId)` MUST return `true`.
```

Comment:
I'd _really_ like to see the interface id actually spelled out explicitly before going to final.